fvm/meter/weighted/meter.go

```diff
@@ -56,6 +54,11 @@ var (
 		common.MemoryKindBoundFunction:           25,
 		common.MemoryKindBigInt:                  50,
 
+		common.MemoryKindAtreeArrayDataSlab:     72,
+		common.MemoryKindAtreeArrayMetaDataSlab: 1024,
+		common.MemoryKindAtreeMapDataSlab:       72,
+		common.MemoryKindAtreeMapMetaDataSlab:   1024,
```

Comment:
@fxamacker I added these weights based on our slack discussion, but let me know if these values seem correct to you.

For context for others, we previously metered length (slabs) as if they consumed 1kb of space. In actuality this value is how much data they can hold, not how much memory require in and of themselves. Instead we should just meter the overhead of the data slabs themselves.

Reply:
@dsainati1 I think the initial estimates for atree slabs are:

```
- atree array DataSlab overhead: 72 bytes
- atree array MetaDataSlab: 1 KiB
- atree map DataSlab overhead: 140 bytes
- atree map element (key+value) overhead: 64 bytes
- atree array MetaDataSlab: 1 KiB 
```

But based on @SupunS findings on Friday, I am going to look into the 720-byte discrepancy for empty struct.  So we probably will tweak the overhead listed above.

Reply:
There is a separate overhead for each element of the map?

Reply:
> There is a separate overhead for each element of the map?

Yes, because we have to store map key's hash digest (and related overhead) for retrieval.

Reply:
Hmm, sounds like we would need another memory kind for this, unless you think we can just fold that overhead into the cost of MapDataSlab?

Reply:
Yeah, I think it might be easier to have another memory kind, especially because we are still tweaking weight for each type.